### ISSUE TITLE
Export result cannot be used to Encrypt (Not compatible)

### ISSUE THREAD

**Issue by HDLP9** [tka85/dotenvenc#1]
Hi,

Just started using the package and its very good and easy to use. Congrats.

I think there is an issue with the export method because if we export some encrypted variable, and tried to encrypt again the env variables with have " and ;

Example:

.env.local

```
VAR_1='secret_test'
VAR_2=like_this
```
and encrypting to .env.local.enc.

If we decrypt .env.local.enc to console we will have this:

```
export VAR_1="secret_test";
export VAR_2="like_this";
```

putting this result in .env.local and encrypting again we will have the following ENVs in nextjs for example:

```
console.log(process.env.VAR_1);
-> result will be "secret_test"; and not only secret_test
```

```
console.log(process.env.VAR_2);
-> result will be "like_this"; and not only like_this
```

I think the issue is in here 

https://github.com/tka85/dotenvenc/blob/64607a4ecf21eca86e4c12b312be287bfc0c568f/src/index.ts#LL66C65-L66C65

If we change this to using ' instead of " and removing the ; we are good to go.

**Comment by tka85** [comment]
Sorry for the delayed response.

Can you please clarify what you did when you say: `putting this result in .env.local and encrypting again...` ?

Printing the decrypted output is meant to be used in combination with `eval` on a shell to setup the environment, like it is stated in the README:

```
eval `./node_modules/.bin/dotenvenc -d`
```

The intention is not to save this output (which contains `export`) and re-encrypt it.

**Comment by mbstuart** [comment]
@tka85 I think the use case that OP has in mind is as follows:

1. Dev A creates a .env file
2. Dev A uses dotenvenc to create a .env.enc file
3. Dev A commits & pushes .env.enc 
4. Dev B pulls the latest code from trunk, including .env.enc
5. Dev B decrypts .env.enc into their local .env file <-- what dotenvenc can't do at the moment
6. Dev B makes some changes to the .env file
7. Dev B re-encrypts and pushes the new .env.enc

The value of this is step 6/7 - the ability for members of the team to make shared changes to a common .env file 

Edit: to be clear I see this as a feature request (for ability to decrypt .env.enc -> .env format)

**Comment by tka85** [comment]
Just published [v5.0.0](https://github.com/tka85/dotenvenc/releases/tag/v5.0.0) which introduces two changes:

* a new `-x` flag for the `dotenvenc` binary which prints on the console the contents of the encrypted secrets as `export` statements. This was until now done with the `-d` flag which rightfully led to the current issue being opened.
* the existing `-d` flag for the `dotenvenc` binary will now print on the console the contents of the encrypted secrets in a format that can be saved into `.env` file. This is useful when you want to recreate locally the unencrypted `.env` (e.g. new team members checkout the project or in case of loss/corruption of `.env`).

I believe this solves the issue.

**Comment by mbstuart** [comment]
awesome! thanks @tka85
